scribble-doc/scribblings/scribble/acmart.scrbl

```diff
@@ -203,6 +203,23 @@ screen version of the image links to the badge authority.
   @abstract{This is an abstract.}
 }|
 
+@defproc[(authorsaddresses [addresses pre-content?] ...) block?]{
+ Sets the text for the authors' addresses on the first page
+ in some styles. By default this field is set to the authors'
+ affiliation.
+
+ The @racket[addresses] parameter is the address text. As a
+ special case the empty list removes the adresses field entirely.
+
+ @codeblock|{
+  #lang scribble/acmart @acmsmall
+  @title{A fancy paper}
+  @author["Leif Andersen"]
```

Comment:
Would probably be more "in-style" with the rest of the docs to use a fictional character.

Reply:
I'm happy to change it. I was just following the David Van Horn from above. Would you like me to change that too?

Reply:
Either way fixed, thanks.